src/main/resources/lang/default.lang

```diff
@@ -405,6 +405,7 @@ chat styles:
 	run_command: run command, command, cmd
 	suggest_command: suggest command, sgt
 	change_page: change page
+	copy_to_clipboard: copy to clipboard, copy, clipboard
```

Comment:
minor but could we add a `ctp` option as well? tince tooltip has `ttp` I figured it makes ense

Reply:
I feel like copy is a better shorthand. `ctp` is confusing to me